src/main/java/grid/Node.java

```diff
@@ -0,0 +1,18 @@
+package grid;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.grid.selenium.GridLauncherV3;
+
+import static utils.DataPropLoader.getNodeCapabilityProperty;
+
+public class Node {
+    // linux java -Dwebdriver.gecko.driver="/home/linuxnode/Downloads/geckodriver" -Dwebdriver.chrome.driver="/home/linuxnode/Downloads/chromedriver" -jar selenium-server-standalone.jar -role node -port 5555 -hub http://192.168.0.105:4444/grid/register
+    public static void main(String[] args) {
+        WebDriverManager.firefoxdriver().setup();
+        WebDriverManager.chromedriver().setup();
+        WebDriverManager.iedriver().setup();
+        GridLauncherV3.main(new String[]{"-role", "node", "-hub",
+                getNodeCapabilityProperty("node.url") + "/grid/register",
```

Comment:
This is not nod URL. It is Hub url. It says that node should be registered for specific Hub by url 

P.S Why not keep full path in properties like hub.register_url ? Instead of make it on the fly